CONTRIBUTING.md

```diff
@@ -130,18 +131,42 @@ BREAKING CHANGE: Description of what broke and how to achieve this behavior now
 
 * Push to a GitHub fork or to a branch (naming convention: `<user>/<feature-bug-name>`)
 * Submit a Pull Requests on GitHub and assign the PR for a review to the "awslabs/aws-cdk" team.
+* Please follow the PR checklist written below. We trust our contributors to self-check, and this helps that process!
 * Discuss review comments and iterate until you get at least one “Approve”. When iterating, push new commits to the
   same branch. Usually all these are going to be squashed when you merge to master. The commit messages should be hints
   for you when you finalize your merge commit message.
 * Make sure to update the PR title/description if things change. The PR title/description are going to be used as the
-  commit title/message and will appear in the CHANGELOG, so maintain them all the way throughout the process.
+  commit title/message and will appear in the CHANGELOG, so maintain them all the way throughout the process.  
 
 ### Step 6: Merge
 
 * Make sure your PR builds successfully (we have CodeBuild setup to automatically build all PRs)
 * Once approved and tested, a maintainer will squash-merge to master and will use your PR title/description as the
   commit message.
 
+---
+### Pull Request Checklist
```

Comment:
This doesn’t feel like it’s in the right place. Should probably go to the step above where you actually submit the PR or the beginning as a quick overview.

Reply:
+1

Reply:
Moved to top!